### DISCUSSION TITLE
Credentials keep duplicating after every update

### DISCUSSION THREAD

**Discussion by BobWs** [victorbalssa/abacus#324] [Bug]
Hi,
I have noticed that after every update or when a update notification pops up my account credentials keep duplicating out of the blue, sometimes two times but sometimes even 4x.
Is any body else having this issue?

**Comment by victorbalssa** [comment]
I can't reproduce it on my phone/tablet, can you share your phone spec? thx

**Reply by BobWs** [reply]
I will as soon as it happens again!

**Comment by killtux** [comment]
i had same problem last year for months, then i deleted all logins and started with a fresh login.
since then it started to duplicated again but i couldnt figure out what causes this.

now i have a full page of logins again and the screen jumps if you have more than 4-5 logins somehow. really crazy.

![image](https://github.com/user-attachments/assets/e8fec5ef-0d93-473b-8593-6a5cf2bd3d77)

iPhone 14pro, happend at iOS17 back then and even on iOS18 now.

**Reply by BobWs** [reply]
Yep, still having this problem, I keep deleting the duplicates but they just reappear after a while. Same problem on both devices (iPhone and iPad). Also tried to delete the app on the iPad to see if that helped, but it didn't.

**Comment by lverissimo** [comment]
Same happens with me and my family.... all iphones. Iphone 15, 12 and 11.

**Reply by victorbalssa** [reply]
It should be fixed with the version `0.21.0`

**Reply by BobWs** [reply]
I’m sorry to report that the issue is still not resolved. 

I continue to experience duplicated login credentials. Additionally, there’s a new problem: sometimes, even when there’s only one login entry in the list, I’m unable to access the app. It seems like the login credentials are getting stuck, and the only way to resolve it is by completely removing the login and re-entering all the details before I can log in again. 

So, unfortunately, the issue persists.

**Reply by victorbalssa** [reply]
Should be fixed with the latest OTA patch `0.21.1`.
Let me know if it works 🙌🏻
